engine/language-server/src/main/scala/org/enso/languageserver/text/CollaborativeBuffer.scala

```diff
@@ -130,6 +130,7 @@ class CollaborativeBuffer(
       stop(Map.empty)
 
     case IOTimeout =>
+      logger.warn("Timeout reached when awaiting file's content")
```

Comment:
Wouldn't be this the right moment to dump all stacktraces or turn on [profiling](https://github.com/enso-org/enso/blob/5f0a16c87c62c6c29e73c1617875bf9b0a4a3d2a/engine/language-server/src/main/scala/org/enso/languageserver/profiling/ProfilingManager.scala#L54)?

Reply:
Not there yet, but yes, probably.